common/web/types/test/lexical-model-types.tests.ts

```diff
@@ -5,19 +5,19 @@
  * imported and compiled without any compiler errors.
  */
 
-import { USVString, Transform, Suggestion, SuggestionTag, Context, Capabilities, Configuration, Distribution, WordBreakingFunction, Span, LexicalModelPunctuation, ElementString, KMXPlus } from '@keymanapp/common-types';
+import { KMXPlus, ElementString, LexicalModelTypes } from "@keymanapp/common-types";
```

Comment:
Is `'@keymanapp/common-types';` the convention used in the other files?

Reply:
single vs double quotes is not important, so I won't change it. We use both extensively.